src/core/default/commands/dynamodb/shell.py

```diff
@@ -0,0 +1,70 @@
+import cmd
+import json
+
+from argparse import ArgumentParser
+from typing import List, Tuple
+
+import aurora_data_api
+from rich.console import Console
+from rich.table import Table
+
+from core.constructs.commands import BaseCommand
+from core.default.commands.dynamodb.utils import get_dynamodb_info_from_cdev_name
+
+import core.default.mappers.aws_client as aws_client
+
+
+class shell(BaseCommand):
+
+    help = """
+        Open an interactive shell to a non-relational db.
+    """
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        parser.add_argument(
+            "resource",
+            type=str,
+            help="The database to execute on. Name must include component name. ex: comp1.myDb",
+        )
+        parser.add_argument(
+            "--put_item", help="put item command from a json in single quotes"
```

Comment:
These should be a single argument with a set of options https://docs.python.org/3/library/argparse.html#choices. It could be called something like 'action'.

Also there is no notes about how the data should be structured to work with the dynamodb api https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html#DynamoDB.Client.get_item. The boto3 api expects the data in a weird shape, so you should make this CLI more ergonomic instead of just passing through the args from the CLI.

Reply:
I've did this way because the choices method doesnt allow a string after the choice